src/Microsoft.ML.Transforms/PermutationFeatureImportanceExtensions.cs

```diff
@@ -205,24 +212,27 @@ public static class PermutationFeatureImportanceExtensions
         /// <param name="features">Feature column name.</param>
         /// <param name="useFeatureWeightFilter">Use features weight to pre-filter features.</param>
         /// <param name="topExamples">Limit the number of examples to evaluate on. null means examples (up to ~ 2 bln) from input will be used.</param>
+        /// <param name="permutationCount">The number of permutations to perform.</param>
         /// <returns>Array of per-feature 'contributions' to the score.</returns>
-        public static ImmutableArray<MultiClassClassifierMetrics>
+        public static ImmutableArray<MultiClassClassifierMetricsStatistics>
             PermutationFeatureImportance(
                 this MulticlassClassificationContext ctx,
                 IPredictionTransformer<IPredictor> model,
                 IDataView data,
                 string label = DefaultColumnNames.Label,
                 string features = DefaultColumnNames.Features,
                 bool useFeatureWeightFilter = false,
-                int? topExamples = null)
+                int? topExamples = null,
+                int permutationCount = 1)
         {
-            return PermutationFeatureImportance<MultiClassClassifierMetrics>.GetImportanceMetricsMatrix(
+            return PermutationFeatureImportance<MultiClassClassifierMetrics, MultiClassClassifierMetricsStatistics>.GetImportanceMetricsMatrix(
                             CatalogUtils.GetEnvironment(ctx),
                             model,
                             data,
                             idv => ctx.Evaluate(idv, label),
                             MulticlassClassificationDelta,
                             features,
+                            permutationCount,
                             useFeatureWeightFilter,
                             topExamples);
         }
```

Comment:
(Also as I can't comment on the exact right line, I'll place slightly above)

It seems odd to call this delta "`MultiClassClassifierMetrics`" as the values are in a different range than the standard metrics. Eg: `accuracyMicro` can be negative. The case for confusion is when a user hovers over their `var` or `MultiClassClassifierMetrics` not realizing it comes from `MulticlassClassificationDelta()` and assumes (by its classname) that it's the actual metric. 

Perhaps we could inherit a class `MultiClassClassifierMetricsDelta` from `MultiClassClassifierMetrics` to distinguish?

https://github.com/dotnet/machinelearning/blob/07b8429b5ca4715575c109a3d8313a3d4bd49c42/src/Microsoft.ML.Transforms/PermutationFeatureImportanceExtensions.cs#L248-L256

Reply:
I agree; however, these are only used privately and the user only gets `MultiClassClassifierMetricsStatistics` objects back. The thought behind this was to minimize the number of objects we have to think about, especially as this is private.